test/Microsoft.OpenApi.Tests/Validations/Visitors/OpenApiVisitorSetTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.OpenApi.Exceptions;
+using Microsoft.OpenApi.Models;
+using Xunit;
+
+namespace Microsoft.OpenApi.Validations.Visitors.Tests
+{
+    public class OpenApiVisitorSetTests
+    {
+        [Fact]
+        public void VisitorsPropertyReturnsTheCorrectVisitorList()
+        {
+            for (int i = 0; i < 5; i++) // 5 is just a magic number
+            {
```

Comment:
Not sure what the loop helps with. Maybe remove?

Reply:
I just want to test multiple retrieving the property "Visitors" can work as expected.

Reply:
I just want to test multiple retrieving the property "Visitors" can work as expected.

---
In reply to: [157649800](https://github.com/Microsoft/OpenAPI.NET/pull/152#discussion_r157649800) [](ancestors = 157649800)